code/modules/antagonists/brainwashing/brainwashing.dm

```diff
@@ -35,6 +35,7 @@
 	show_name_in_check_antagonists = TRUE
 	ui_name = "AntagInfoBrainwashed"
 	suicide_cry = "FOR... SOMEONE!!"
+	replace_banned = TRUE
```

Comment:
![image](https://user-images.githubusercontent.com/24975989/148698376-a4f3c395-0d2b-4469-a93b-0f7c33e336de.png)

replace_banned is already `TRUE` at the `/datum/antagonist` level, so this won't do anything different.

Have a look at `/datum/antagonist/proc/is_banned(mob/M)` and where it's called from, then make sure that `/datum/antagonist/brainwashed` has all the correct vars set to actually trigger the checks in `/datum/antagonist/proc/on_gain()`.

Also look at `/proc/brainwash(mob/living/L, directives)` as part of this, since this appears to be the primary proc involved in giving people the brainwashed antag datum.

If the issue report is accurate, somewhere in that loop of `player gets antag > on_gain() > is_banned() > replace player with ghost` will be your bug.

There linked issue is also from 2020. There is a possibility that it's simply not broken anymore. I'm sure any admin would be happy to help test this on the live servers, since it relates to their tools working properly and they'd have to be awfully mean to refuse.